docs/debops-playbooks/guides/linux-containers.rst

```diff
@@ -9,24 +9,27 @@ Using Linux containers
 Host requirements
 -----------------
 
-**Your host must be Debian based**
+**Your host must be Debian based.**
 
-It can be Ubuntu 14.x, Debian Wheezy/Jessie, etc.
+Debian stable and oldstable are supported by debops, as well as the two latest
+LTS releases of Ubuntu. However the main focus will always be towards the
+latest stable/LTS release. So if you plan on setting up a new machine, always
+use the latest release.
```

Comment:
You could write "DebOps" instead of "debops" to better match the rest of the docs.

"However the main focus will always be towards the latest stable/LTS release" → "However the main focus will always be towards the latest Debian Stable followed by the latest Ubuntu LTS release"